src/devices/Fsr408/Fsr408.cs

```diff
@@ -0,0 +1,154 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Diagnostics;
+
+namespace Iot.Device.Fsr408
+{
+    public class Fsr408 : IDisposable
+    {
+        private GpioController _controller;
+        private int _pinNumber = 18;
+
+        public Mcp3008.Mcp3008 AdcConverter { get; set; }
+        public int Resistance { get; set; } = 10000; // 10k ohm
+        public int PowerSupplied { get; set; } = 5000;  // 5 mV
+
+        public Fsr408()
+        {
+            _controller = new GpioController();
+            _controller.OpenPin(_pinNumber);
+        }
+        public Fsr408(int pinNumber)
+        {
+            _pinNumber = pinNumber;
+            _controller = new GpioController();
+            _controller.OpenPin(_pinNumber);
+        }
+
+
+        public int ReadFromMcp3008()
+        {
+            if (AdcConverter != null)
+            {
+                return AdcConverter.Read(0);
+            }
+            else
+            {
+                throw new NotSupportedException("ADC converter not set");
+            }
+        }
+
+        public int ReadCapacitorChargingDuration()
```

Comment:
I'd consider making this private but if it remains public please at least add Ms/Milliseconds suffix